optout/src/main/java/gov/cms/ab2d/optout/OptOutS3.java

```diff
@@ -0,0 +1,84 @@
+package gov.cms.ab2d.optout;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.iterable.S3Objects;
+import com.amazonaws.services.s3.model.*;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+
+public class OptOutS3 implements Runnable {
```

Comment:
Small comment, non-blocking. I recommend taking advantage of Lombok if you'd like. I believe this project uses it already in different classes. You could add these annotations: @NoArgsConstructor, @RequiredArgsConstructor, @AllArgsConstructor, and @Builder for this class and take advantage of them. Feel free to ignore though! just a style choice

https://projectlombok.org/features/constructor
https://projectlombok.org/features/Builder

Reply:
Thank you for your comment!
We use lombok in the ab2d repository, but we do not use it in ab2d-lambdas and even exclude it from dependencies when we import something from AB2D-Libs.
The Lombok library increases the jar size and AWS has a size quota for lambda. In other words, we want to import only the necessary libraries.

Reply:
Makes sense and thank you for explaining, @smirnovaae!